doc/src/site/sphinx/migration_guide.rst

```diff
@@ -21,6 +21,10 @@ From 3.30 to 3.32
 - The ``setH2OCluster(ip, port)`` method on ``H2OConf`` in all APIs doesn't implicitly set the external backend anymore.
   The method ``setExternalClusterMode()`` must be called explicitly.
 
+- The method ``DLModel`` in ``water.support.DeepLearningSupport`` is removed. Please use Sparkling Water algorithm API
```

Comment:
I would maybe mention that it's related only to Scala API.

Reply:
👍